Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Controller.purs

```diff
@@ -1762,6 +1767,16 @@ eval MapReadyAction state = continueWithCmd state [ do
 eval (TriggerPermissionFlow flowType) state = exit $ ExitToPermissionFlow flowType
 
 
+eval (AppUpdateBanner (Banner.OnClick)) state = do
+  if os == "IOS" then do
+      continueWithCmd state [do
+        _ <- pure $ printLog "App Link" (getValueFromConfig "APP_LINK")
+        _ <- pure $ openAppStore (getValueFromConfig "appId")
```

Comment:
We can also create a common function instead of two and handle the call in JBridge